Dockerfile

```diff
@@ -34,7 +34,7 @@ WORKDIR /app
 
 ENV APP_ENV=prod
 
-RUN mkdir var && \
+RUN mkdir -p data/articles var && \
     chown www-data:www-data var
```

Comment:
`data/articles` is created by `root` (I think, this is before `USER www-data`) so won't have ownership by `www-data` unless chown is made `-r`ecursive